In musl libc there is no struct mallinfo and no function mallinf()

--- qtwebengine/src/3rdparty/chromium/content/child/content_child_helpers.cc	2015-06-05 04:52:18.502230985 +0200
+++ qtwebengine/src/3rdparty/chromium/content/child/content_child_helpers.cc	2015-06-05 04:52:06.712231020 +0200
@@ -15,7 +15,7 @@

 namespace content {

-#if defined(OS_LINUX) || defined(OS_ANDROID)
+#if (defined(OS_LINUX) && defined(__GLIBC__)) || defined(OS_ANDROID)
 size_t GetMemoryUsageKB() {
   struct mallinfo minfo = mallinfo();
   uint64_t mem_usage =
